subworkflows/local/alevin.nf

```diff
@@ -31,9 +31,9 @@ workflow SCRNASEQ_ALEVIN {
         """Must provide a genome fasta file ('--fasta') and a gtf file ('--gtf'), or a genome fasta file
         and a transcriptome fasta file ('--transcript_fasta`) if no index is given!""".stripIndent()
 
-    if (transcript_fasta) {
+    if (transcript_fasta || salmon_index) {
```

Comment:
Actually, maybe it would be neater if this whole conditional was deleted, and this was just  part of the assertions above?

```    assert (salmon_index && txp2gene) || (genome_fasta && gtf) || (genome_fasta && transcript_fasta && txp2gene):
        """Must provide a genome fasta file ('--fasta') and a gtf file ('--gtf'), or a genome fasta file
        and a transcriptome fasta file ('--transcript_fasta`) if no index and txp2gene is given!""".stripIndent()
```

Reply:
I agree! Done.